bloom-housing/bloom
# 1928/adding-mfa-to-partners-portal

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/1928)


## Description
This adds MFA to the partners portal

## How Can This Be Tested/Reviewed?
Step 1: create a user that has mfaEnabled (there is currently no way to set this via the front end, but creating a new user should do it) 
Step 2: attempt to log in as that user, you should have to go through the new MFA steps
Step 3: in order to test this with an sms you will need to talk to @seanmalbert to get your phone number added as a verified test number


## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
